modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -15853,13 +16192,25 @@ private void _testPostValidate(
 
 			_setUpPermissionThreadLocal(user);
 
+			String errorScopeKey;
+
+			if (Validator.isNumber(scopeKey)) {
+				errorScopeKey = (scopeKey != null) ? scopeKey : "";
+			}
+			else {
+				errorScopeKey = String.valueOf(
+					GroupUtil.getGroupId(
+						TestPropsValues.getCompanyId(), scopeKey,
+						_groupLocalService));
+			}
```

Comment:
Check `Validator.isNumber(scopeKey)` implementation and you will see that it already handles null value.

```
String errorScopeKey = scopeKey;

if (!Validator.isNumber(scopeKey)) {
      errorScopeKey = String.valueOf(
	      GroupUtil.getGroupId(
		      TestPropsValues.getCompanyId(), scopeKey,
		      _groupLocalService));
}
```

what do you think?